cluster/ci/monitoring/mixins/grafana_dashboards/build_cop.jsonnet

```diff
@@ -95,176 +124,44 @@ dashboard.new(
-        min='0',
-        max='1',
-        formatY1='percentunit',
-    ) + legendConfig)
-    .addTarget(prometheus.target(
-        'sum(rate(prowjob_state_transitions{job="plank",job_name=~"branch-.*-images",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state="success"}[${range}]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"branch-.*-images",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state=~"success|failure"}[${range}]))',
-        legendFormat='branch-.*-images',
-    ))
-    .addTarget(prometheus.target(
-        'sum(rate(prowjob_state_transitions{job="plank",job_name=~"release-.*-4.1",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state="success"}[${range}]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"release-.*-4.1",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state=~"success|failure"}[${range}]))',
-        legendFormat='release-.*-4.1',
-    ))
-    .addTarget(prometheus.target(
-        'sum(rate(prowjob_state_transitions{job="plank",job_name=~"release-.*-4.2",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state="success"}[${range}]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"release-.*-4.2",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state=~"success|failure"}[${range}]))',
-        legendFormat='release-.*-4.2',
-    ))
-    .addTarget(prometheus.target(
-        'sum(rate(prowjob_state_transitions{job="plank",job_name=~"release-.*-upgrade.*",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state="success"}[${range}]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"release-.*-upgrade.*",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state=~"success|failure"}[${range}]))',
-        legendFormat='release-.*-upgrade.*',
-    ))
-    .addTarget(prometheus.target(
-        'sum(rate(prowjob_state_transitions{job="plank",job_name=~"release-.*4.1.*4.2.*",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state="success"}[${range}]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"release-.*4.1.*4.2.*",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state=~"success|failure"}[${range}]))',
-        legendFormat='release-.*4.1.*4.2.*',
-    )), gridPos={
-    h: 9,
-    w: 24,
-    x: 0,
-    y: 0,
-  })
+    myPanel('Job Success Rates for github.com/${org}/${repo}@${base_ref}',
+        description='sum(rate(prowjob_state_transitions{job="plank",job_name=~"<job_name_expr>",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state="success"}[${range}]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"<job_name_expr>",job_name!~"rehearse.*",org=~"${org}",repo=~"${repo}",base_ref=~"${base_ref}",state=~"success|failure"}[${range}]))'
```

Comment:
Could we factor this out into a helper method as well and share it with the prometheus target func, or just write human descriptions here? Once we upgrade to the newer grafana we will not want these since we can just go to data source links, right?

Reply:
Changed to human descriptions.
Also tested with 6.3.3 grafana ... The datalinks (6.3+) work fine.
I found that the queries in the descriptions cannot be rendered well with 6.3.3.
Maybe it is telling us not to use queries as description (although it is i believe a UI bug).